sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -117,14 +117,14 @@ def _get_digest_from_tag(self, repository: str, tag: str) -> str:
         return tag_props.digest
 
     @distributed_trace
-    def delete_repository(self, repository: str, **kwargs) -> None:
+    def delete_repository(self, repository: str, **kwargs: Any) -> None:
         """Delete a repository. If the repository cannot be found or a response status code of
         404 is returned an error will not be raised.
 
         :param str repository: The repository to delete
         :returns: None
         :rtype: None
-        :raises: ~azure.core.exceptions.HttpResponseError
+        :raises: :class:`azure.core.exceptions.HttpResponseError`
```

Comment:
raises acts slightly different than rtype/returns, the colon goes at the end:

```suggestion
        :raises ~azure.core.exceptions.HttpResponseError:
```